README.md

```diff
@@ -1,11 +1,20 @@
 # User guides [![Build Status](https://travis-ci.org/actix/book.svg?branch=master)](https://travis-ci.org/actix/book) [![Join the chat at https://gitter.im/actix/actix](https://badges.gitter.im/actix/actix.svg)](https://gitter.im/actix/actix?utm_source=badge&utm_medium=badge&utm_campaign=pr-badge&utm_content=badge)
 
-* [Actix User Guide](https://actix.rs/book/actix/)
-* [Actix API Documentation (Development)](https://actix.rs/actix/actix/)
-* [Actix API Documentation (Releases)](https://docs.rs/actix/)
+- [Actix User Guide](https://actix.rs/book/actix/)
+- [Actix API Documentation (Development)](https://actix.rs/actix/actix/)
+- [Actix API Documentation (Releases)](https://docs.rs/actix/)
 
-* [Actix Web User Guide](https://actix.rs/docs/)
-* [Actix Web API Documentation (Development)](https://actix.rs/actix-web/actix_web/)
-* [Actix Web API Documentation (Releases)](https://docs.rs/actix-web/)
+- [Actix Web User Guide](https://actix.rs/docs/)
+- [Actix Web API Documentation (Development)](https://actix.rs/actix-web/actix_web/)
+- [Actix Web API Documentation (Releases)](https://docs.rs/actix-web/)
 
-* [Chat on gitter](https://gitter.im/actix/actix)
+- [Chat on gitter](https://gitter.im/actix/actix)
+
+## Using this library
+
+This repository is an [`mdBook`](https://github.com/rust-lang-nursery/mdBook)
+project. To use it for this project:
+
+- Install `mdBook` if you haven't already: `cargo install mdbook`
+- In the `actix` directory: `mdbook watch -o`
+  - This automatically opens your browser and watches the md files for changing
```

Comment:
I don't think you need nested list `This automatically` just move it outer sentence?

Reply:
I don't know how much it matters but I like to think the outer list is a list of instructions and the inner lists are explanations. I don't really need a list really I guess.

Reply:
Regardless I should also clarify you'll still need to refresh the page

Reply:
Ok, let me know once you'll clarify it and I'll merge

Reply:
Clarified. I kept it in the inner list. Let me know if you want me to change that.